pages/common/cowsay.md

```diff
@@ -1,24 +1,28 @@
 # cowsay
 
-> Generate an ASCII character (by default a cow) saying or thinking something.
+> Print an ASCII friend (by default a cow) saying or thinking something.
 > More information: <https://github.com/tnalpgge/rank-amateur-cowsay>.
 
-- Print an ASCII cow saying "Hello world":
+- Print an ASCII cow saying "Hello World":
 
-`cowsay "Hello world"`
+`cowsay "{{Hello World}}"`
```

Comment:
I dont like the title case in this sentence.

Reply:
What would you have it as then?
Isn't the top usage meant to be the most basic, which will be built off from to have more advanced commands.

Reply:
I'm not sure how much I like it either, now that you bring it up.  However, it seems to be more common across the pages (32 vs. 13) 🤔

Reply:
Probably `Hello world`.

Reply:
Deleted my post since it was irrelevant to what was being asked...

Well I feel if we are going to change this one, then the rest need to be changed

Reply:
I personally believe it should be one of the following

> Hello World
hello world
Hello, World
Hello, World!

Reply:
I like the last one the best.

Reply:
I suggest the following grammar:
```suggestion
`cowsay "{{hello, world}}"`
```

The reasoning for this is that the [first known instance of a hello world](https://en.wikipedia.org/wiki/%22Hello,_World!%22_program#History) uses this exact casing and grammar.

Reply:
Should I apply that throughout?